laws/shared/src/test/scala/algebra/laws/LawTests.scala

```diff
@@ -123,6 +123,8 @@ class LawTests extends munit.DisciplineSuite {
   checkAll("Long", RingLaws[Long].commutativeRing)
   checkAll("Long", LatticeLaws[Long].boundedDistributiveLattice)
 
+//  catsKernelStdOrderForBigInt
+  checkAll("BigInt", OrderLaws[BigInt].truncatedDivision)
```

Comment:
Can we add implementations for the specialized things?

Reply:
Same comment as for the intermediate ring structures-- the instances are only approximations due to the type limited range.